src/Console/Helper/ResultHelper.php

```diff
@@ -37,11 +37,18 @@ public function sort(array $result): array
         /** @var StopwatchEvent $event */
         foreach ($result as $vendor => $event) {
             $memory = sprintf('%.2F', $event->getMemory() / 1024 / 1024);
```

Comment:
The memory should be an average also, do you agree?

Reply:
all in all, memory metrics is quite broken. Total memory consumption rises with each `BenchmarkSample->run()`. The results will be different between single-vendor benchmark and all-vendors benchmarks. I don't think there's a good way to measure memory consumption in all-vendors run.

Reply:
The memory metrics should probably be either dropped from reporting or thoroughly thought through and redesigned to better reflect serializer footprints.
Probably, we should drop the `--exclude` flag in favor of some `--vendor` flag, which would allow to run only selected vendor. In this single-vendor mode the memory footprint is more trustworthy and might be shown in overall statistics.

Reply:
Ok, I've tried running a batch of 1000 objects with 1, 10, 100 and 1000 samples, outputting the `$event->getMemory()` as is, in bytes. And every time I get the same exact value `8388608`, which means samples count does not affect memory consumption and therefore should not be used to calculate average memory consumption.

Reply:
@tsantos84 please, review once again. Memory consumption cannot be averaged just like that with division by samples count

Reply:
@eduard-sukharev, so the final memory value will be the total amount of memory considering all samples? If yes, it's ok to me to keep the memory in the result view, otherwise I believe that we should display that metric only when running the benchmark with 1 sample.

Reply:
@tsantos84 it seems that final memory value does not depend on samples count, which means if we attempt to average it dividing by samples count it will reduce proportionally to the samples count.
So, for now, it seems to me that we either should either leave it as is or remove it at all if it's considered unreliable.

Reply:
@eduard-sukharev ok, lets remove this metric for now until we find a reliable solution to display it.